src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,76 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final String COMMA_SPLITTER = ",";
+    private static final String SPACE = " ";
+    private static final String SUPPLY = "supply";
+    private static final String BUY = "buy";
+    private static final String RESULT = "result";
+    private static final String LINE_SEPARATOR = System.lineSeparator();
+    private static final String WRITE_EXCEPTION = "Can't write to file %s";
+    private static final String READ_EXCEPTION = "Can't read file with name %s";
+    private final int[] types = new int[2];
```

Comment:
[Do you really need this array as a class-level variable? Better create it where you need to use it ](https://mate-academy.github.io/jv-program-common-mistakes/java-core/builder-file/work-with-file#local-variables)